openlibrary/macros/BookPreview.html

```diff
@@ -1,11 +1,12 @@
 $def with (ocaid, linkback=True)
 $# :param str ocaid:
-<div class="cta-button-group">
-  <a class="cta-btn cta-btn--shell cta-btn--preview"
-     data-iframe-src="https://archive.org/details/$ocaid?view=theater&wrapper=false"
-     data-iframe-link="https://archive.org/details/$ocaid"
-     data-ol-link-track="CTAClick|Preview" href="#bookPreview">$_('Preview')</a>
-</div>
+
+  <div class="cta-button-group">
+    <a class="cta-btn--preview"
```

Comment:
```suggestion
    <a class="cta-btn cta-btn--preview"
```
As discussed, the preview button styling isn't being applied to carousel buttons:
![Screenshot from 2023-05-04 09-26-53](https://user-images.githubusercontent.com/28732543/236350128-908cdb56-f256-422e-a5ef-d19d41418fdd.png)

I've got an idea that may fix this.  Adding the `cta-btn` class will make this look like a button in carousels again, but the button's background will be grey.  Changing the background color of `cta-btn--preview` buttons to white should fix it.

Would doing something like this make it more difficult to properly position the preview link on the books page?

Reply:
@jimchamp do you know if the same style of "Preview" link is desired in the carousel? (book icon, blue text link)

I'm starting to implement these changes, but seems like it might look a little out of place if the other CTAs in the carousel are rectangular buttons:
<img width="1098" alt="Screen Shot 2023-05-04 at 9 22 44 PM" src="https://user-images.githubusercontent.com/13765801/236377227-1efe5843-fb68-4413-b2ac-e99052a0b29f.png">

To that end, do we forsee this new style of "Preview" link having widespread use throughout the site? Or is it more of a Book/Edition-specific look? This will impact how I structure the CSS -- apologies, I didn't realize that my updates changed it in this area, and I just want to make sure I think through the right way to implement.

We could potentially have a variation here, maybe something like this:
<img width="1096" alt="Screen Shot 2023-05-04 at 9 26 36 PM" src="https://user-images.githubusercontent.com/13765801/236377605-68e73ace-7ce5-404f-ace7-ef64bdb894ff.png">

Chat with you about this tomorrow  🙂 👍